lib/benchmarks/src/Cardano/Wallet/Benchmarks/Collect.hs

```diff
@@ -180,6 +183,43 @@ showUnits MegaBytes = "MB"
 showUnits GigaBytes = "GB"
 showUnits Count = "count"
 
+convertUnits :: Units -> Units -> Double -> Double
```

Comment:
```suggestion
convertUnits :: Unit -> Unit -> Double -> Double
```

I'm referring to code that has already been merged, but I would recommend to name this data type to `Unit` (singular) instead of `Units`. In this way, the statement `Seconds :: Unit` can be read out load as "`Seconds` are a `unit` of time" which makes perfect grammatical sense.

Perhaps in a separate PR.